src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,36 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
+import { dayJS } from '@/app/services/dayjs';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
```

Comment:
I think it's better to call it `type` since we're talking about an online/offline event or course

Reply:
It's named 'mode' in course card and in data also. If i'll change it only here it will be inconsistent.